src/main/java/jalau/at18/architects/GreenCard.java

```diff
@@ -4,8 +4,8 @@ public class GreenCard extends Card {
     private String color;
```

Comment:
Can we move common attributes like `color` to `Card` asbtract class? they should be also set as `protected` so they are accessible from subclasses